spec/controllers/students_controller_spec.rb

```diff
@@ -1,16 +1,18 @@
 require 'rails_helper'
 
 RSpec.describe StudentsController, :type => :controller do
+  let(:student) { FactoryBot.create(:student, { first_name: "John", last_name: "Smith" }) }
```

Comment:
I reckon you don't need to explicitly declare brackets:
```suggestion
  let(:student) { FactoryBot.create(:student, first_name: "John", last_name: "Smith") }
```

Reply:
Fixed